src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/CulturePreservingExecutionContext.cs

```diff
@@ -281,7 +284,7 @@ public void Dispose()
         private class CultureAndContextManager
         {
             #region Constructor 
-
+            [SecurityCritical]
```

Comment:
Why are we adding CAS annotations in this PR? I thought the plan was to eventually get rid of them all.

Reply:
This is a work in progress port from some ideas that were kicking around in .NET Framework for a fix.

We moved this off to .NET 5 and @arpitmathur is far from done with this in multiple ways.  I wouldn't look at this as anything of note at this point.